core/src/main/java/org/jruby/ir/targets/IRBytecodeAdapter6.java

```diff
@@ -589,9 +589,9 @@ public void invokeOtherOneFloat(String file, int line, String name, double flote
         if (!MethodIndex.hasFastFloatOps(name)) {
             pushFloat(flote);
             if (callType == CallType.NORMAL) {
-                invokeOther(file, line, name, 1, false, false);
+                invokeOther(file, line, name, 1, false, false, false);
```

Comment:
Hmmm I wonder how many people static final booleans for this sort of issue?  I don't think I have ever seen that done but it is probably a pattern?